src/styles.css

```diff
@@ -173,14 +241,14 @@ button {
 }
 
 .featured__main-title {
-  top: 100px;
+  top: 150px;
   color: #011936;
   font-family: 'Quicksand';
   font-size: 85px;
   font-weight: 700;
-  position: absolute;
+  position: fixed;
   right: 40px;
-  z-index: 1;
+  z-index: -1;
```

Comment:
what is this doing - just curious!!

Reply:
Since we had to change the main title to be in fixed position, it would overlap with our tagged recipes. I changed the z-index to -1 so it would appear behind the tagged recipes container.